tests/analysis.spec.ts

```diff
@@ -119,6 +119,7 @@ describe('Functional test of analysis', () => {
           source,
           bundleHash: bundle.fileBundle.bundleHash,
           files: [],
+          base64Encoding: false
```

Comment:
Would be cool to have test for ` base64Encoding: true` case

Reply:
Agreed - I don't think I can do this until the `deeproxy` changes are made live though.

Reply:
Not sure how the structure of tests is here, but generally, for a unit test you can.

You can check if the request contains encoded payloads. We don't care what deeproxy response with (we just want to test the code changes of code-client)

Reply:
Unit tests added